### DISCUSSION TITLE
Modpack Mod List?

### DISCUSSION THREAD

**Discussion by richaell** [Blasto33/Minecraft-Steam-Deck-Edition#4] [General]
Saw that the modlist is on the "In Progress" board. Any updates on it? Id love to see what I will be installing before doing so. Curious to see compatibility with Better Minecraft modpack.

**Comment by Blasto33** [comment]
Hello,

Thanks for your interest in my modpack! You'll find the modlist on Modrinth, right here: https://modrinth.com/modpack/minecraft-steam-deck-edition/version/0.5.0

I'm working on the update for the latest version but it takes time :) Feel free to report any issue!
